engine/src/main/scala/cromwell/webservice/WorkflowSubmitBatchActor.scala

```diff
@@ -35,9 +34,7 @@ case class WorkflowManagerBatchIndexedResponse(response: WorkflowManagerResponse
 class WorkflowSubmitIndexedActor(batchSubmitActor: ActorRef, requestHandlerActor: ActorRef,
                                  source: WorkflowSourceFiles, index: Int) extends Actor {
 
-  override def preStart() = {
-    requestHandlerActor ! OldStyleWorkflowManagerActor.SubmitWorkflow(source)
-  }
+  override def preStart() = ???
```

Comment:
huh?  If this is something PBE Cromwell needs to support I don't recall seeing a ticket for it.